help/en/html/hardware/arduino/Sidebar.shtml

```diff
@@ -1,12 +1,12 @@
 <!-- Sidebar.shtml -->
-<!-- This is the common sidebar definition for html/hardware/[subdirectory]/ pages -->
+<!-- Il s'agit de la d&#233;finition commune de la barre lat&#233;rale pour les pages html/hardware/[sous-r&#233;pertoire]/ -->
```

Comment:
Why are English comments, here and several other places in this file, being changed to some language other than English, in a file that is in the English branch of help?